x/group/keeper/keeper.go

```diff
@@ -230,3 +240,61 @@ func (k Keeper) MaxMetadataLength() uint64 { return k.config.MaxMetadataLen }
 func (k Keeper) GetGroupSequence(ctx sdk.Context) uint64 {
 	return k.groupTable.Sequence().CurVal(ctx.KVStore(k.key))
 }
+
+func (k Keeper) iterateProposalsByVPEnd(ctx sdk.Context, cb func(proposal group.Proposal) (stop bool)) error {
```

Comment:
what do you think of: 
```suggestion
func (k Keeper) iterateProposalsByVPEnd(ctx sdk.Context, cb func(proposal group.Proposal) (bool, error)) error {
```

so that we allow the callback to error:

```go
		stop, err := cb(proposal)
		if err != nil {
			return err
		}
		if stop {
			break
		}
```

I don't like the fact that right now, if the callback errors, we just break (and commit the store changes before break). We should definitely throw an error altogether.